asdf_astropy/converters/fits/fitswcs.py

```diff
@@ -0,0 +1,30 @@
+from asdf.extension import Converter
+
+
+class FitsWCSConverter(Converter):
+    """
+    Converter for serializing and deserializing `astropy.wcs.WCS` objects.
+
+    This converter currently supports the serialization of simple WCS objects
+    by preserving the `wcs.to_header()` data. It does not support complex WCS objects
```

Comment:
```suggestion
    by preserving the ``wcs.to_header()`` data. It does not support complex WCS objects
```
This should hopefully fix the docs.

Reply:
Since we are now using `to_fits` instead of `to_header`, we no longer need to specify that the converter supports only certain types of `WCSes`.